_docs/concepts/security/authn-policy.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Istio Authentication Policy
+overview: Describes Istio authentication policy
+
+order: 10
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+## Overview
+Istio authentication policy enables operators to specify authentication requirements for a service (or services). Istio authentication policy is composed of two parts authentication:
+
+* Peer: verifies the party, the direct client, that makes the connection. The common authentication mechanism for this is [mutual TLS]({{home}}/docs/concepts/security/mutual-tls.html). Istio is responsible for managing both client and server sides to enforce the policy.
+
+* Origin: verifies the party, the original client, that makes the request (e.g end-users, devices etc). JWT is the only supported mechanism for origin authentication at the moment. Istio configures the server side to perform authentication, but doesn't enforce that the client side sends the required token.
+
+
+Identities from both authentication parts, if applicable, are output to the next layer (e.g authorization, Mixer). To simplify the authorization rules, the policy can also specify which identity (peer or origin) should be used as 'the principal'. By default, it is set to the peer's identity.
+
+
+## Architecture
+
+Authentication policies are saved in Istio config store (in 0.7, the storage implementation uses Kubernetes CRD), and distributed by control plane. Depending on the size of the mesh, configs propagation may take a few seconds to a few minutes. During the transition, it might expect traffic lost or inconsistent authentication results.
```

Comment:
it might -> you can expect

This sentence worries me. Is this implying that if I'm going from valid state A to valid state B, some requests may be allowed through that shouldn't be because the system is in some sort of transient state?

Reply:
Right. Authn policy rollout is something we need to address next. This is required some features from v2 (http/https multiplex, port routing rule etc). 

Also, inconsistency mainly because not all replicas get updates at the same time. Per instance, it is consistent.